CHANGELOG.md

```diff
@@ -10,6 +10,23 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 -->
 <!--Start Of Releases (DO NOT DELETE THIS LINE)-->
 
+## [1.4.5] - 2022-12-18
+
+## Highlights
+
+This is a bugfix release, primarily for diff-quality. In addition, a new rules for spacing
+around parenthesis is also included.
```

Comment:
Think it's worth mentioning the 2.0 plan, and that this may be the last 1.x release?

Reply:
Good idea